admin_guide/limits.adoc

```diff
@@ -103,38 +108,46 @@ Per container, the following must hold true if specified:
 |Constraint |Behavior
 
 |`*Min*`
-|Min[resource] <= container.resources.requests[resource] (required) <= container/resources.limits[resource] (optional)
+|`Min[resource]` less than or equal to `container.resources.requests[resource]`
+(required) less than or equal to `container/resources.limits[resource]`
+(optional)
 
 If the configuration defines a `min` CPU, then the request value must be greater
 than the CPU value. A limit value does not need to be specified.
 
 |`*Max*`
-|container.resources.limits[resource] (required) <= Max[resource]
+|`container.resources.limits[resource]` (required) less than or equal to
+`Max[resource]`
 
 If the configuration defines a `max` CPU, then you do not need to define a
 request value, but a limit value does need to be set that satisfies the maximum
 CPU constraint.
 
 |`*MaxLimitRequestRatio*`
-|MaxLimitRequestRatio[resource] <= ( container.resources.limits[resource] / container.resources.requests[resource])
+|`MaxLimitRequestRatio[resource]` less than or equal to (
+`container.resources.limits[resource]` /
+`container.resources.requests[resource]`)
```

Comment:
(There may be other instances of this elsewhere; i'll refrain from commenting on them individually.)